### ISSUE TITLE
BootFailedException when numeric contains a period for decimal in some locales

### ISSUE THREAD

**Issue by rhealefever** [umbraco/Umbraco-CMS#13711]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

11.1.0

### Bug summary

For some locales, Umbraco is unable to boot after installing a package (or transferring content via Deploy or uSync) with a Numeric property editor with periods in the decimal place.

### Specifics

```
Umbraco.Cms.Core.Exceptions.BootFailedException: Boot failed: Umbraco cannot run. See Umbraco's log file for more details.

-> Umbraco.Cms.Core.Exceptions.BootFailedException: The input string '199.99' was not in a correct format.

-> System.FormatException: The input string '199.99' was not in a correct format.
   at System.Number.ThrowOverflowOrFormatException(ParsingStatus status, ReadOnlySpan`1 value, TypeCode type)
   at System.Convert.ToDecimal(String value, IFormatProvider provider)
   at System.Convert.ChangeType(Object value, Type conversionType, IFormatProvider provider)
   at NPoco.MappingHelper.<>c__DisplayClass2_0.<GetConverter>b__6(Object src)
   at NPoco.RowMappers.PropertyMapper.MapValue(GroupResult`1 posName, Object[] values, Func`2 converter, Object instance, PocoColumn pocoColumn, Object defaultValue)
   at NPoco.RowMappers.PropertyMapper.<>c__DisplayClass8_2.<BuildMapPlans>b__3(DbDataReader reader, Object[] values, Object instance)
   at NPoco.RowMappers.PropertyMapper.<>c__DisplayClass7_0.<BuildMapPlan>b__1(DbDataReader reader, Object[] values, Object instance)
   at NPoco.RowMappers.PropertyMapper.Map(DbDataReader dataReader, RowMapperContext context)
   at NPoco.MappingFactory.Map(DbDataReader dataReader, Object instance)
   at NPoco.Database.Read[T](Object instance, DbDataReader r, PocoData pd)+MoveNext()
   at NPoco.Database.QueryImp[T](T instance, Expression`1 listExpression, Func`2 idFunc, Sql Sql, PocoData pocoData)+MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at NPoco.Database.Fetch[T](Sql sql)
   at Umbraco.Extensions.NPocoDatabaseExtensions.<>c__DisplayClass12_0`2.<FetchByGroups>b__0(IEnumerable`1 x)
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
   at Umbraco.Extensions.EnumerableExtensions.SelectByGroups[TResult,TSource](IEnumerable`1 source, Func`2 selector, Int32 groupSize)+MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at Umbraco.Cms.Infrastructure.Persistence.Repositories.Implement.ContentRepositoryBase`3.GetPropertyCollections[T](List`1 temps)
   at Umbraco.Cms.Infrastructure.Persistence.Repositories.Implement.DocumentRepository.MapDtosToContent(List`1 dtos, Boolean withCache, Boolean loadProperties, Boolean loadTemplates, Boolean loadVariants)
   at Umbraco.Cms.Infrastructure.Persistence.Repositories.Implement.DocumentRepository.<GetPage>b__56_0(List`1 x)
   at Umbraco.Cms.Infrastructure.Persistence.Repositories.Implement.ContentRepositoryBase`3.GetPage[TDto](IQuery`1 query, Int64 pageIndex, Int32 pageSize, Int64& totalRecords, Func`2 mapDtos, Sql`1 filter, Ordering ordering)
   at Umbraco.Cms.Infrastructure.Persistence.Repositories.Implement.DocumentRepository.GetPage(IQuery`1 query, Int64 pageIndex, Int32 pageSize, Int64& totalRecords, IQuery`1 filter, Ordering ordering)
   at Umbraco.Cms.Core.Services.ContentService.GetPagedLocked(IQuery`1 query, Int64 pageIndex, Int32 pageSize, Int64& totalChildren, IQuery`1 filter, Ordering ordering)
   at Umbraco.Cms.Core.Services.ContentService.GetPagedDescendants(Int32 id, Int64 pageIndex, Int32 pageSize, Int64& totalChildren, IQuery`1 filter, Ordering ordering)
   at Umbraco.Cms.Core.Services.ContentService.SaveAndPublishBranch(IContent document, Boolean force, Func`2 shouldPublish, Func`4 publishCultures, Int32 userId)
   at Umbraco.Cms.Core.Services.ContentService.SaveAndPublishBranch(IContent content, Boolean force, String culture, Int32 userId)
   at UmbTrainingSite.Package.Migrations.PublishRootBranchPostMigration.Migrate()
   at Umbraco.Cms.Infrastructure.Migrations.MigrationBase.Run()
   at Umbraco.Cms.Infrastructure.Migrations.MigrationPlanExecutor.Execute(MigrationPlan plan, String fromState)
   at Umbraco.Cms.Infrastructure.Migrations.Upgrade.Upgrader.Execute(IMigrationPlanExecutor migrationPlanExecutor, ICoreScopeProvider scopeProvider, IKeyValueService keyValueService)
   at Umbraco.Cms.Infrastructure.Install.PackageMigrationRunner.RunPackagePlans(IEnumerable`1 plansToRun)
   at Umbraco.Cms.Infrastructure.Install.UnattendedUpgrader.HandleAsync(RuntimeUnattendedUpgradeNotification notification, CancellationToken cancellationToken)
   at Umbraco.Cms.Core.Exceptions.BootFailedException.Rethrow(BootFailedException bootFailedException)
   at Umbraco.Cms.Web.Common.Middleware.BootFailedMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass6_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at SixLabors.ImageSharp.Web.Middleware.ImageSharpMiddleware.Invoke(HttpContext httpContext, Boolean retry)
   at Umbraco.Forms.Web.HttpModules.ProtectFormUploadRequestsMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass6_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
```

### Steps to reproduce

Change default locale on machine (works on Windows, Mac, Linux) to a language that uses commas (,) instead of periods (.) as decimals for prices (For example, 100,05 instead of 100.05).

Install a starter kit which has a Numeric property editor that has a period as a decimal, like UmbTrainingSite.Package.

Try to run the project. Umbraco will not be able to boot.

### Expected result / actual result

Expected result:
Umbraco boots. This has always been the case until this version.

Actual result:
Umbraco does not boot.

**Comment by Jay-umbr** [comment]
Some of the people who had this issue suspect it might be due to the initial database generation. They would create a new project, install the content package mentioned above and run into the problem.

**Comment by elit0451** [comment]
Locally, I had no problem booting Umbraco 11.1 with the **UmbTrainingSite.Package** even though my `DefaultUILanguage` was set to `"da-DK"`. 

One thing to point out - it seems like the decimal value that is inside a Decimal property editor (_and not a Numeric one_) with the following configuration:

![image](https://user-images.githubusercontent.com/21998037/213717568-7993fd4e-612e-4e96-826b-4955e14d7476.png)

**Comment by Zeegaan** [comment]
I finally managed to reproduce this!! 
My steps

- Go to language and region settings in windows 
![image](https://user-images.githubusercontent.com/70372949/214266470-2ca31b35-fa86-40fb-b2d5-7e00f28bb5a8.png)
- Install french and use it as windows display language, delete the other language WARNING DO NOT DO THIS IF YOU ARE NOT A SEASONED USER, note to use numbers on the french keyboard, either use the numpad or hold SHIFT ( I found out the hard way)
- Restart your computer
- open the umbraco project and set your DefaultUILanguage to french in AppSettings.json. like so
![image](https://user-images.githubusercontent.com/70372949/214266916-e7ba9f81-d34d-48cd-8547-31c684e0a6b0.png)

**Comment by nikolajlauridsen** [comment]
This has been fixed in #13741, thanks for reporting 😄
